Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.yml

```diff
@@ -763,57 +963,95 @@ script:
+      type: string
+    description: Command gives the status of the running task based on the task id.
     name: trendmicro-visionone-check-task-status
     polling: true
-    description: Command gives the status of the running task based on the task id.
   - arguments:
     - description: ID of the workbench you would like to attach the note to.
       name: workbench_id
       required: true
-    - description: Contents of the note to be attached
+    - description: Contents of the note to be attached.
       name: content
       required: true
     outputs:
-    - contextPath: VisionOne.Add_Note.Workbench_Id
-      description: The ID of the workbench that the note was added to.
+    - contextPath: VisionOne.Add_Note.code
+      description: HTTPS status code of making the request.
       type: string
-    - contextPath: VisionOne.Add_Note.note_id
-      description: The ID of the note that was added.
+    - contextPath: VisionOne.Add_Note.message
+      description: Message notifying the user of note added to workbench.
       type: string
-    - contextPath: VisionOne.Add_Note.code
-      description: The response code from the command
+    - contextPath: VisionOne.Add_Note.note_id
+      description: ID of the note added to workbench.
       type: string
-    description: Attaches a note to a workbench alert
+    description: Attaches a note to a workbench alert.
     name: trendmicro-visionone-add-note
   - arguments:
     - description: ID of the workbench you would like to update the status for.
       name: workbench_id
       required: true
-    - description: Status to assign to the workbench alert
+    - description: Target resource will be updated only if it matches ETag of the target one. Etag is one of the outputs from get_alert_details.
+      name: if_match
+      required: true
+    - description: Status to assign to the workbench alert.
       name: status
       required: true
       auto: PREDEFINED
       predefined:
       - new
-      - in progress
-      - true positive
-      - false positive
+      - in_progress
+      - true_positive
+      - false_positive
+      - benign_true_positive
+      - closed
     outputs:
     - contextPath: VisionOne.Update_Status.Workbench_Id
       description: The ID of the workbench that had the status updated.
       type: string
     - contextPath: VisionOne.Update_Status.code
-      description: The response code from the command
+      description: HTTP status code of updating workbench alert status.
+      type: string
+    - contextPath: VisionOne.Update_Status.message
+      description: Message notifying user that the alert status has been updated to user defined status.
       type: string
-    description: Updates the status of a workbench alert
+    description: Updates the status of a workbench alert.
     name: trendmicro-visionone-update-status
-  dockerimage: demisto/python3:3.10.12.63474
+  dockerimage: devdemisto/pytmv1-061:1.0.0.77283
```

Comment:
please fix the comments in https://github.com/demisto/dockerfiles/pull/20932 and after it will be merged we should put here the right image